docs/api.md

```diff
@@ -67,6 +67,10 @@ packager({
 })
 ```
 
+**NOTE:** `afterCopy` will not be called if `prebuiltAsar` is set.
+
+---
```

Comment:
The lines are unnecessary.

Reply:
I thought it was a nice visual grouping. I'll drop them.

Reply:
I think if we're going to do that, it should be in a separate PR.